float32 gripper_position
float32[] arm_joints

# enforce a user manual override to run behavior (may not be enforced)
bool manual_safety